app/controllers/merchant/dashboard_controller.rb

```diff
@@ -2,7 +2,27 @@ class Merchant::DashboardController < Merchant::BaseController
   before_action :require_merchant
 
   def index
+    @pending_orders = []
```

Comment:
Incredible work. I don't think there is anything wrong with how we attempted this

Reply:
Attempt 1: tried writing a model method on Orders
Attempt 2: tried brainstorming a model method on Merchants
Attempt 3: nuked and tried creating a joins table for orders and merchants
Attempt 4: SUCCESS! @LHJE and this incredible work. I feel like we could track this pattern and interpret our ActiveRecord association of merchant_id back to order_id more easily now! Looking forward to refactoring this with you down the road!

Reply:
Thanks pal!  Yeah, it'll be nice to make this AR... one day